OptoTap/README.md

```diff
@@ -42,5 +49,67 @@ You can safely connect voltages from 5V to 24V to this board.
   - changed C1 to higher voltage rating
   - changed C2 bootstrap cap to higher capacitance
   - changed all capacitors to 0805 for better part availability
+- 2.4: updated release
+  - added polyfuse to limit overcurrent and damp inrush current-induced voltage spike on hot-plugging
+  - added reverse polarity protection diode 
+  - increased size of inductor to prevent coil saturation
+  - added a second output filter capacitor to reduce output voltage ripple
+  - hybrid footprint to permit use of EE-SX398 sensor in addition to Optek versions
+  - two-color LED to indicate status and assist with troubleshooting.
+  - 'Dark Mode' solder jumper to allow turning off indicator LEDs
+  - added 5V/24V selection solder jumper on rear of board to allow for easy bypassing of voltage regulator circuit
+  - increased thermal relief on solder pads to make through-holes easier to solder
+
+
+## Indicator LEDs
+
+The revised boards (V1.3/2.4) now have two-color indicator LEDs to help assist with troubleshooting. This should hopefully help a little bit with troubleshooting. The expected behavior of the LED is:
+
+- Blue: Connected to power, beam not broken
+- Red: Connected to power, beam broken
+
+<img src="./images/led_blink.gif" width=480>
+
+When installed in the toolhead, the light should be red normally (toolhead in 'down' position) and blue when activated (toolhead in 'up') position.
+
+I would suggest powering on the board with 5V prior to installing it to make sure everything is functioning as expected. If the indicator LED is red with nothing blocking the optical sensor beam path, there is probably a problem with the ground and signal wires -- triple check!
+
+
+## Sensor Choice
+
+
+### Omron or Optek
+
+The revised versions of the boards (V1.3 and V2.4) feature a hybrid optical sensor footprint that permits installation of the Omron EE-SX398 sensor in addition to the Optek OPB666/9xx versions. In testing, the Omron sensor performed just as well as the Optek sensor. 
+
+However, it has a *slightly* different footprint than the Omron sensor, with annoyingly-close leads on the three pin side. I went through a few iterations of testing to come up with a hybrid footprint that seems to work well for either sensor model.
+
+The slots mean there is a little play in sensor positioning, particularly with the EE-SX398 sensor. To help ensure precise alignment with this sensor, print out the jig included in the [STLs]('./STLs') folder.
```

Comment:
Link incorrect (./STLs). Should be an absolute link (/STLs)

Reply:
haha you commented as I was pushing the change :D It seems to work now in my branch.